.github/workflows/shared-release.yml

```diff
@@ -21,8 +21,18 @@ env:
   MODULE_NAME_CAPITALIZED: ${{ inputs.module_directory == 'web/bible-on-site' && 'Website' || 'API' }}
   DOCKER_IMAGE_NAME: ${{ inputs.module_directory == 'web/bible-on-site' && 'bible-on-site' || 'bible-on-site-api' }}
   DOCKER_IMAGE_PATH: ${{ inputs.module_directory }}/.release/${{ inputs.module_directory == 'web/bible-on-site' && 'bible-on-site' || 'bible-on-site-api' }}.tar.gz
+
 jobs:
+  setup_env:
```

Comment:
makes no sense to add this as a separate job, this might have broken CI, it's best to use steps, making too many jobs is adding overhead